push/tests/integration/sample_client.cc

```diff
@@ -23,7 +23,7 @@ static std::string GetHostName() {
   return hostname;
 }
 
-int main(int argc, char** argv) {
```

Comment:
Why remove the 2 parameters?
I think it's unnecessary to do this change.

Reply:
argc and argv are not used in the function body.

Reply:
But for unused parameters, we should not remove them directly.
Just use `IGNORE` to ignore them.